packages/fxa-settings/src/components/Banner/index.tsx

```diff
@@ -144,7 +144,7 @@ export const ResendCodeSuccessBanner = ({
 
   const content = {
     localizedHeading: ftlMsgResolver.getMsg(
-      'resend-code-success-heading',
+      'resend-code-success-banner-heading',
```

Comment:
Can we do this a separate polish PR? I want to make sure that these MFA PRs are trivial to rollback.